packages/react-theme-provider/src/useThemeTokenClass.ts

```diff
@@ -0,0 +1,29 @@
+import { useWindow } from '@fluentui/react-window-provider';
+import * as React from 'react';
+import { ThemeProviderState } from './ThemeProvider.types';
+import { tokensToStyleObject } from './tokensToStyleObject';
+
+/**
+ * Give the tokens defined within the theme, create a className
+ * to append to the className on state. Memoizes the result to
+ * avoid re-computations.
+ */
+export const useThemeTokenClass = (state: ThemeProviderState) => {
```

Comment:
call this `useThemeProviderClass` instead?

Reply:
This is specifically for the tokens in theme; not the `ThemeProvider` classes, which is separate and uses `makeClasses`.

Reply:
hmmm, actually, this could move into `useThemeProviderClass`

Reply:
resolved.